public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchLogsQueryRunner.ts

```diff
@@ -295,58 +280,16 @@ export class CloudWatchLogsQueryRunner extends CloudWatchRequest {
       })),
     };
 
-    if (options.makeReplacements) {
```

Comment:
what did this do before?

Reply:
That's a good question, and also the main driver behind this refactoring! 
As said before, all different query subTypes called this method. Some of these sub types wanted their query props to be replaced as variablas. If so, they set the `makeReplacements` arg to true. 

If you look below, what happens is pretty nasty with a deeply nested if statements. The idea here is that variables are supposed to be interpolated before the backend query is issued. For the logs query, it's in the `handleLogQueries` method above. And all the resource requests that used to be in this file are now moved to api.ts ts and that's also where the variables are being interpoalted.